doc/multiprecision.qbk

```diff
@@ -2149,15 +2194,20 @@ or GCC-6 or later in C++14 mode.
 Compilers other than GCC and without `std::is_constant_evaluated()` will support a very limited set of operations:
 expect to hit roadblocks rather easily.
 
+See __compiler_support for __is_constant_evaluated;
+
 For example given:
 
 [constexpr_circle]
 
-We can now calculate areas and circumferences using all constexpr arithmetic:
+We can now calculate areas and circumferences, using all compile-time `constexpr` arithmetic:
 
 [constexpr_circle_usage]
 
-Note that these make use of the numeric constants from the Math library, which also happen to be `constexpr`.
+Note that these make use of the numeric constants from the __math_constants library, which also happen to be `constexpr`. 
+These usually have the full precision of the floating-point type, here 128-bit, about 36 decimal digits.
```

Comment:
Hopefully, they should always have the full precision of the FT type?

Reply:
These should always have the full precision of the floating-point type, here 128-bit, about 36 decimal digits.